### ISSUE TITLE
SE 4.01 using 100% of a single thread while idle

### ISSUE THREAD

**Issue by tariq0101** [SubtitleEdit/subtitleedit#7556]
SE 4.01 using 100% of a single thread while idle even without loading a subtitle file or a video.
I'm on Linux mint debian and using mono on the portable version.
it doesn't use the same thread it hops from to another but always using 13% of the cpu for no apparent reason, draining my laptop's battery and making it hot.

**Comment by niksedk** [comment]
I think more info is needed...

**Comment by tariq0101** [comment]
> I think more info is needed...

I'll gladly provide that. can you tell me what you need and how to get it?

**Comment by niksedk** [comment]
Sorry, I cannot re-create this:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/38b9b150-f515-4cc6-9c35-360750cbd788)

**Comment by niksedk** [comment]
Could you provide some screenshots?

**Comment by tariq0101** [comment]
> Could you provide some screenshots from Linux mint?

Hi, the issue resolved itself after a reboot, sorry for wasting your time.
however there's another issue I'm facing when try to transcribe using whisper-ctranslate2 
whisper can't find the temporary wav file although I've manually checked and it exists and I have the appropriate permissions for it.
![Screenshot from 2023-10-25 19-31-58](https://github.com/SubtitleEdit/subtitleedit/assets/59182266/97a38986-efb7-4cc2-b124-4298e159e98d)

**Comment by niksedk** [comment]
> whisper-ctranslate2

What happens if you type `whisper-ctranslate2` in a terminal window?

**Comment by Purfview** [comment]
@tariq0101 Try to use Faster-Whisper, it has Linux binary.

**Comment by tariq0101** [comment]
> @tariq0101 Try Faster-Whisper, it has Linux binary.

it was removed as far as I can tell.
I can only choose original whisper, cpp, ctranslate2 and WhisperX.
the same issue happens on WhisperX.

**Comment by Purfview** [comment]
> it was removed as far as I can tell. I can only choose original whisper, cpp, ctranslate2 and WhisperX...

Hmm, @niksedk can you add it?

**Comment by tariq0101** [comment]
> > whisper-ctranslate2
> 
> What happens if you type `whisper-ctranslate2` in a terminal window?


it runs normally since I installed it before SubtitleEdit
the problem is that whisper (any version) can's find the wav file but it exists, it's really strange.
this is the output:

`You need to specify one or more audio files
Use `whisper-ctranslate2 --help` to see the available options.
`

**Comment by niksedk** [comment]
@tariq0101: try checking the file `whisper_log.txt` (you can also press F2 in the Whisper window, to see cli output)
Perhaps SE is using an old/wrong cli parameter...

**Comment by tariq0101** [comment]
> @tariq0101: try checking the file `whisper_log.txt` (you can also press F2 in the Whisper window, to see cli output) Perhaps SE is using an old/wrong cli parameter...

I copied the command from `whisper_log.txt` and paste it in the terminal and it ran smoothly, there's nothing wrong with the command itself.
of course I did that before closing SE so the temp file was there.
is there a recommended directory where I should place SE portable folder?

I can't see the cli output with F2 because it crashes almost immediately after pressing "generate"
